contributing/deprecation_policy.md

```diff
@@ -38,51 +128,70 @@ A three-month deprecation is similar with more time for clients to adjust code:
 
 Map this schedule onto reality: 
 
-1. Is there a holiday/conference coming up that would prevent clients from being able to collaborate and comment? 
+1. Is there a holiday/conference coming up that would prevent clients from being able to
+   collaborate and comment? 
 1. Is the to-be-replaced code particularly problematic for some reason? 
 1. Does the deprecation fall near a release of the operating system or new devices?
 
-## Do the work
+**Once you have determined a schedule, add the schedule to the migration guide**.
 
-Before announcing the deprecation, create an issue in GitHub that represents it. That issue will stay open until the deletion PR is submitted.
+### Step 6: Announce that the API will be deprecated
 
-TODO: Include announcement instructions. Generally, this is done via blog.
+Inform internal clients of the impending deprecation by sending an email with the following
+format:
 
-Deprecate APIs using the macro provided by Apple in AvailabilityMacros.h that includes a message:
-* DEPRECATED_MSG_ATTRIBUTE(msg)
+```
+Subject: YYY will eventually be deprecated.
+Body:
+YYY will eventually be deprecated. Please see <link to migration guide> for more details.
 
-This macro requires an explanation string which should be similar to: ”This API will be removed on D/M/YYYY. Please use XYZ instead. See issue #n for more details.”
+The timeline for this deprecation is as follows:
 
-For any code that should be eventually deleted, add the following comment nearby to make it easier to find and remove the code: // DEPRECATION(github.com/material-components/material-components-ios/issues/n)
+- <Deprecation date>: The API will be formally marked as deprecated. You will start receiving
+  build warnings if you are still using the deprecated API.
+- <Deletion date>: The API will be deleted.
+```
 
-Once you submit the breaking PR, don’t immediately pile on subsequent PRs that would conflict with an emergency rollback.
+### Step 7: After the deprecation date, deprecate the API
 
-## Versioning
+Send a pull request annotating the API as deprecated. Use `__deprecated_msg("")` and provide a
+concise message with a recommendation for an alternative API, if any is available.
 
-Adding the deprecation *warning* (no breaking change) is a minor version bump.
+### Step 8: After the deletion date, re-evaluate usage of the API
 
-Actually changing the code (breaking change) is a major version bump.
+Once the deletion date has passed and at least one release has been cut since the API was marked as
+deprecated, you can now consider deleting the API. But first: you must evaluate whether the API is
```

Comment:
Do you mean to wait after the API is marked "as ready for deletion" - not "as deprecated"?

Reply:
We only annotate APIs in two manners in this proposal:

- First, as "to be deprecated".
- Second, as "deprecated".

We won't annotate APIs as ready for deletion because we'll simply delete them once they're ready to be deleted.